deeplearning4j/deeplearning4j-core/src/test/java/org/deeplearning4j/nn/layers/HelperUtilsTest.java

```diff
@@ -83,8 +82,9 @@ public void testOneDnnHelperCreation() {
                 LocalResponseNormalizationHelper.class,"layername",getDataType()));
         assertNotNull(HelperUtils.createHelper("", MKLDNNSubsamplingHelper.class.getName(),
                 SubsamplingHelper.class,"layername",getDataType()));
-        assertNotNull(HelperUtils.createHelper("", "",
-                DropoutHelper.class,"layername",getDataType()));
```

Comment:
Did you remove that intentionally?

Reply:
No.  Let me revert that change.

Reply:
Actually, this did need to be removed for cpu yes.